packages/typespec-test/test/contentsafety_modular/generated/typespec-ts/src/api/operations.ts

```diff
@@ -0,0 +1,502 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  AnalyzeTextResult,
+  ImageData,
+  AnalyzeImageResult,
+  TextBlocklist,
+  TextBlockItemInfo,
+  AddOrUpdateBlockItemsResult,
+  TextBlockItem,
+  TextBlocklistResult,
+  TextBlockItemListResult
+} from "../models/models.js";
+import {
+  isUnexpected,
+  ContentSafetyContext as Client,
+  AddOrUpdateBlockItems200Response,
+  AddOrUpdateBlockItemsDefaultResponse,
+  AnalyzeImage200Response,
+  AnalyzeImageDefaultResponse,
+  AnalyzeText200Response,
+  AnalyzeTextDefaultResponse,
+  CreateOrUpdateTextBlocklist200Response,
+  CreateOrUpdateTextBlocklist201Response,
+  CreateOrUpdateTextBlocklistDefaultResponse,
+  DeleteTextBlocklist204Response,
+  DeleteTextBlocklistDefaultResponse,
+  GetTextBlocklist200Response,
+  GetTextBlocklistDefaultResponse,
+  GetTextBlocklistItem200Response,
+  GetTextBlocklistItemDefaultResponse,
+  ListTextBlocklistItems200Response,
+  ListTextBlocklistItemsDefaultResponse,
+  ListTextBlocklists200Response,
+  ListTextBlocklistsDefaultResponse,
+  RemoveBlockItems204Response,
+  RemoveBlockItemsDefaultResponse,
+} from "../rest/index.js";
+import {
+  StreamableMethod,
+  operationOptionsToRequestParameters,
+} from "@azure-rest/core-client";
+import { uint8ArrayToString } from "@azure/core-util";
+import {
+  GetTextBlocklistOptions,
+  CreateOrUpdateTextBlocklistOptions,
+  DeleteTextBlocklistOptions,
+  ListTextBlocklistsOptions,
+  GetTextBlocklistItemOptions,
+  ListTextBlocklistItemsOptions,
+} from "../models/options.js";
+import { AddOrUpdateBlockItemsOptionsOptions, AnalyzeImageOptionsOptions, AnalyzeTextOptionsOptions, RemoveBlockItemsOptionsOptions } from "../models/index.js";
+
+export function _analyzeTextSend(
+  context: Client,
+  text: string,
+  options: AnalyzeTextOptionsOptions = { requestOptions: {} }
+): StreamableMethod<AnalyzeText200Response | AnalyzeTextDefaultResponse> {
+  return context
+    .path("/text:analyze")
+    .post({
+      ...operationOptionsToRequestParameters(options),
+      body: {
+        text: text,
+        categories: options?.categories,
+        blocklistNames: options?.blocklistNames,
+        breakByBlocklists: options?.breakByBlocklists,
+        outputType: options?.outputType,
```

Comment:
it should be `outputType: options?.outputType ?? "FourLevels"` if we choose to respect the default value.